Dockerfile

```diff
@@ -97,6 +97,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         libexif-dev \
         libffi-dev \
         libfontconfig1 \
+        libgbm-dev \
```

Comment:
As per - https://github.com/actions/virtual-environments/issues/732#issuecomment-617764716 - we might want to replace that with `libgbm1`?

Reply:
```suggestion
        libgbm1 \
```

Reply:
Yes, I was wondering. Let me try it. :+1:

Reply:
This works!
Manual testing: [with the fix](https://app.netlify.com/sites/mick/deploys/61423555a71cbd000895a15b#L442), [without the fix](https://app.netlify.com/sites/mick/deploys/6142393d353eae1f904c3068#L423).